src/main/java/at/ac/tuwien/kr/alpha/config/CommandLineParser.java

```diff
@@ -109,19 +111,15 @@
 			.desc("use counting grid normalization instead of sorting circuit for #count (default: " + SystemConfig.DEFAULT_USE_NORMALIZATION_GRID + ")")
 			.build();
 	private static final Option OPT_NO_NOGOOD_DELETION = Option.builder("dnd").longOpt("disableNoGoodDeletion")
-			.desc("disable the deletion of (learned, little active) nogoods (default: "
```

Comment:
Please revert to the old formatting here.

Reply:
done